public/data/qwel-exe.json

```diff
@@ -2,22 +2,27 @@
   "name": "Anirudh Anup",
   "type": "personal",
   "bio": "A passionate 13-year-old developer from India. I am always open to collaborating on projects",
-  "avatar": "https://github.com/qwe123coder.png",
+  "avatar": "https://github.com/qwel-exe.png",
   "links": [
     {
-      "name": "Follow me on GitHub",
-      "url": "https://github.com/qwe123coder",
+      "name": "Follow Me On GitHub",
+      "url": "https://github.com/qwel-exe",
       "icon": "github"
     },
     {
-      "name": "Follow me on Polywork",
+      "name": "Follow Me On Polywork",
       "url": "https://www.polywork.com/anirudh_anup",
       "icon": "globe"
     },
     {
-      "name": "Read my Blog",
+      "name": "Read My Blog",
       "url": "https://dev.to/qwe123coder",
-      "icon": "book"
+      "icon": "DEV.to"
+    },
+    {
+      "name": "Check Out My portfolio",
+      "url": "https://qwel-exe.github.io",
+      "icon": "Globe"
```

Comment:
Shouldn't this be lower-case?

Reply:
@Panquesito7 case doesn’t matter, there’s still a function for that in the app

Reply:
Still, its better to keep everything the same case, etc

Reply:
@qwel-exe , please merge this https://github.com/qwel-exe/LinkFree/pull/3